--- a/.vite/build/workspace-root-drop-handler-DYf1cfzL.js
+++ b/.vite/build/workspace-root-drop-handler-DYf1cfzL.js
@@ -40080,6 +40080,7 @@
   return (
     lz({
+      enabled: !1,
       dsn: t.Si,
       environment: kz.buildFlavor,
       release: i,
       dist: kz.buildNumber ?? void 0,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -71727,6 +71727,7 @@
   let n = GU(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (Sk({
+    enabled: !1,
     dsn: wG,
     environment: e.buildFlavor,
     release: TG(n.version),
     dist: e.buildNumber ?? void 0,
