--- a/webview/assets/settings-back-route-BfYwS6rz.js
+++ b/webview/assets/settings-back-route-BfYwS6rz.js
@@ -25,7 +25,12 @@
   m(`toggleSidebar`, r);
   let i;
   (e[2] === t ? (i = e[3]) : ((i = [t]), (e[2] = t), (e[3] = i)),
-    s(`toggle-sidebar`, r, i));
+    s(`toggle-sidebar`, r, i),
+    v.useEffect(() => {
+      window.__ELECTRON_SHIM__.closeSidebar = () => {
+        u(t, !1);
+      };
+    }, [t]));
 }
 var v = e(t(), 1),
   y = n();
